.github/workflows/train_and_run.yml

```diff
@@ -28,11 +28,19 @@ jobs:
         run: |
           conda activate gecko
           pytest geckoml/models_test.py
-      - name: Run the gecko train script
+      - name: Run the gecko train script for the MLP model
         run: |
           conda activate gecko
           python applications/train_gecko_emulators.py -c config/ci_config.yml
-      - name: Run the gecko box model script
+      - name: Run the gecko box MLP model script
         run: |
           conda activate gecko
           python applications/run_gecko_emulators.py -c config/ci_config.yml
+        - name: Run the gecko train script for the GRU model
```

Comment:
Looks like there is improper YAML syntax here that prevented the CI from running.

Reply:
The sad/funny thing is I watched this fail but was already out the door and onto break 😂

will fix then merge!